routes/files.test.js

```diff
@@ -1,12 +1,17 @@
 const setup = require('./__helpers__/setup');
 
 describe('files routes', () => {
-  const service = { getPageImages: jest.fn() };
+  const files = { getPageImages: jest.fn() };
+  const services = { files };
+  const filesMock = [
+    { file: 'File:image.jpg', url: 'https://en.wikipedia.org/wiki/File:image.jpg' },
+  ];
 
   let context;
 
   beforeEach(async () => {
-    context = await setup({ service });
+    files.getPageImages.mockReset();
```

Comment:
I personally would have intuitively placed the reset in the `afterEach`.
The result should be the same though so… 🤷‍♂️